Sources/Auth+Rx.swift

```diff
@@ -43,7 +43,15 @@ extension Reactive where Base: Auth {
 extension Reactive where Base: Auth {
     public func createUser(withEmail email: String, password: String) -> Single<User> {
```

Comment:
It should change return type from `Single<User>` to `Single<AuthDataResult>`.
By doing so, it looks like the following

```swift
    public func createUser(withEmail email: String, password: String) -> Single<AuthDataResult> {
        return .create { observer in
            self.base.createUser(withEmail: email, password: password, completion: singleEventHandler(observer))
            return Disposables.create()
        }
    }

```

`signInAnonymously`, `signIn(withEmail:password:)` and `signInWith(customToken:` also should change as same.